web/modules/custom/dpl_event/dpl_event.module

```diff
@@ -195,6 +195,24 @@ function dpl_event_preprocess_eventseries(array &$variables): void {
   $variables['formatted_date'] =
     \Drupal::service('dpl_event.reoccurring_date_formatter')
       ->getSeriesDateString($event_series);
+
+  $event_series_instance_count = $event_series->getInstanceCount();
+  if (!$event_series_instance_count > 1) {
+    return;
+  }
```

Comment:
```suggestion
  if ($event_series->getInstanceCount() < 2) {
    return;
  }
```

As I see it there is no reason to keep `event_series_instance_count` in a separate variable.